ios/bottom-tabs/host/RNSBottomTabsHostComponentView.mm

```diff
@@ -368,40 +389,26 @@ - (void)mountingTransactionDidMount:(const facebook::react::MountingTransaction
 
 #pragma mark - LEGACY RCTComponent protocol
 
-RNS_IGNORE_SUPER_CALL_BEGIN
 - (void)insertReactSubview:(UIView *)subview atIndex:(NSInteger)index
 {
-  RCTAssert(
-      [subview isKindOfClass:RNSBottomTabsScreenComponentView.class],
-      @"BottomTabsView only accepts children of type BottomTabScreen. Attempted to mount %@",
-      subview);
-
-  auto *childScreen = static_cast<RNSBottomTabsScreenComponentView *>(subview);
-  childScreen.reactSuperview = self;
-
-  [_reactSubviews insertObject:childScreen atIndex:index];
+  [super insertReactSubview:subview atIndex:index];
+  [self validateAndHandleReactSubview:subview didMount:YES];
+  [_reactSubviews insertObject:subview atIndex:index];
```

Comment:
Also why isn't this call to insert / remove object from `_reactSubview` handled inside the callback?